elasticsearch_metrics/management/commands/sync_metrics.py

```diff
@@ -1,12 +1,37 @@
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand, CommandError
+
+from elasticsearch_metrics.registry import registry
 
 
 class Command(BaseCommand):
     help = "Ensures index templates exist for all Metric classes."
 
     def add_arguments(self, parser):
-        pass
+        parser.add_argument(
+            "app_label",
+            nargs="?",
+            help="App label of an application to synchronize the state.",
+        )
+        # TODO: "using"
 
     def handle(self, *args, **options):
-        # TODO
+        if options["app_label"]:
+            if options["app_label"] not in registry.all_metrics:
+                raise CommandError(
+                    "No metrics found for app '{}'".format(options["app_label"])
+                )
+            app_labels = [options["app_label"]]
+        else:
+            app_labels = registry.all_metrics.keys()
+        for app_label in app_labels:
+            self.stdout.write("Syncing metrics for app '{}'".format(app_label))
+            metrics = registry.get_metrics(app_label=app_label)
```

Comment:
Is there ever a case where we might just want to sync a single metric, or no, since we can just sync all of them?

Reply:
I thought of adding support for single metrics but decided against it in order to keep the code simple. I can't think of a use case for it.